lib/services/users.pm

```diff
@@ -178,7 +178,8 @@ sub full_users_check {
         logout_and_login;
         # for poo#88247, it is hard to deal with the authorization of bernhard in
         # following migration process, we have to restore current user's password.
-        record_soft_failure("poo#88247, it is hard to deal with the authorization of bernhard in following migration process, we have to restore current users password");
+	# We don't need to record softfail here any more since we changed test 
```

Comment:
As you have deleted soft failure at here, it is very strange for other to read this comment

Reply:
Ok, updated.

Reply:
After removing the record_soft_failure of poo#88247, there is no info about poo#88247 any longer, so need to remove its' comments together?

Reply:
I think reserve this comments for poo#88247 to explain why need restore password here is reasonable:
   restore_passwd;